README.md

```diff
@@ -2,24 +2,25 @@
 
 Betaflight is flight controller software (firmware) used to fly multi-rotor craft and fixed wing craft.
 
-This fork differs from baseflight and cleanflight in that it focuses on flight performance and wide target support.
+This fork differs from Baseflight and Cleanflight in that it focuses on flight performance, leading-edge feature additions, and wide target support.
 
 ## Features
 
 Betaflight has the following features:
 
-* Multi-color RGB LED Strip support (each LED can be a different color using variable length WS2811 Addressable RGB strips - use for Orientation Indicators, Low Battery Warning, Flight Mode Status, etc)
-* DSHOT (600, 300 and 150), Oneshot (125 and 42) and Multishot motor protocol support 
-* Blackbox flight recorder logging (to onboard flash or external SD card - where fitted)
+* Multi-color RGB LED strip support (each LED can be a different color using variable length WS2811 Addressable RGB strips - use for Orientation Indicators, Low Battery Warning, Flight Mode Status, Initilization Troubleshooting, etc)
+* DShot (600, 300 and 150), Multishot, and Oneshot (125 and 42) motor protocol support
```

Comment:
Technically it is spelled correctly with the "z" if you're from Canada (which I am) or the USA. But if you're from Britain it's spelled with an "s". ;)

http://www.lukemastin.com/testing/spelling/cgi-bin/database.cgi?action=view_category&database=spelling&category=I

- Travis